rb/lib/selenium/webdriver/common/socket_lock.rb

```diff
@@ -75,9 +70,8 @@ def can_lock?
       end
 
       def did_lock?
-        !!@server
+        @server
```

Comment:
I never really understood the reason to avoid `!!`, but let's remove it if you want.